core/arch/arm/mm/tee_pager.c

```diff
@@ -224,7 +224,7 @@ static void pager_unlock(uint32_t exceptions)
 	cpu_spin_unlock_xrestore(&pager_spinlock, exceptions);
 }
 
-void *tee_pager_phys_to_virt(paddr_t pa)
+void *tee_pager_phys_to_virt(paddr_t pa, size_t len __unused)
```

Comment:
I believe it would be more correct to check that each page covered by `pa .. pa + len - 1` is mapped since the caller should be able to depend on `phys_to_virt()` to be accurate and not just best effort.

A simplified version might be to fail for all ranges spanning more than one page. If that works I think that should be preferable.